ios/Plugin/CapacitorUpdater.swift

```diff
@@ -555,75 +529,183 @@ extension CustomError: LocalizedError {
-                        try self.deleteFolder(source: fileURL)
-                        self.notifyDownload(id, 100)
-                    } catch {
-                        print("\(self.TAG) download unzip error", error)
-                        mainError = error as NSError
-                    }
-                case let .failure(error):
-                    print("\(self.TAG) download error", response.value ?? "", error)
-                    if let afError = error as? AFError,
-                       case .sessionTaskFailed(let urlError as URLError) = afError,
-                       urlError.code == .cannotWriteToFile {
-                        self.sendStats(action: "low_mem_fail", versionName: version)
-                    }
-                    mainError = error as NSError
-                }
+        saveDownloadInfo(version)
+        var checksum = ""
+        var targetSize = -1
+        var totalReceivedBytes: Int64 = loadDownloadProgress() //Retrieving the amount of already downloaded data if exist, defined at 0 otherwise
+         let requestHeaders: HTTPHeaders = ["Range": "bytes=\(totalReceivedBytes)-"]
+        //Opening connection for streaming the bytes
+        AF.streamRequest(url, headers: requestHeaders).validate().onHTTPResponse(perform: { response  in
+            if let contentLength = response.headers.value(for: "Content-Length") {
+                targetSize = (Int(contentLength) ?? -1) + Int(totalReceivedBytes)
             }
-            semaphore.signal()
-        })
-        self.saveBundleInfo(id: id, bundle: BundleInfo(id: id, version: version, status: BundleStatus.DOWNLOADING, downloaded: Date(), checksum: checksum))
-        self.notifyDownload(id, 0)
-        let reachabilityManager = NetworkReachabilityManager()
-        reachabilityManager?.startListening { status in
-            switch status {
-            case .notReachable:
-                // Stop the download request if the network is not reachable
-                request.cancel()
-                mainError = NSError(domain: NSURLErrorDomain, code: NSURLErrorNotConnectedToInternet, userInfo: nil)
+        }).responseStream { [weak self] streamResponse in
+             guard let self = self else { return }
+
+             switch streamResponse.event {
+                 
+             case .stream(let result):
+                 if case .success(let data) = result {
+                     self.tempData.append(data)
+                     
+                     self.savePartialData(startingAt: UInt64(totalReceivedBytes)) // Saving the received data in the package.tmp file
+                     totalReceivedBytes += Int64(data.count)
+                     
+                     let percent = Int((Double(totalReceivedBytes) / Double(targetSize)) * 100.0)
+                     print("Downloading : \(percent)%")
```

Comment:
I believe that this is a debug log. I like it, but I would like it to consistent between IOS and Android.

You can do the following:
1. Remove it
2. Add a `\(self.TAG)` and add a similar log to android